modules/tinymce/changelog.txt

```diff
@@ -2,6 +2,7 @@ Version 5.6.0 (TBD)
     Added new `format_empty_lines` setting to control if empty lines are formatted in a ranged selection #TINY-6483
     Added new user interface `enable` and `disable` methods #TINY-6397
     Added new `closest` formatter API to get the closest matching selection format from a set of formats. #TINY-6479
+    Added new `name` field to the style_select setting object to enable specifying a name for the format. #TINY-4239
```

Comment:
The `style_select` setting doesn't exist? I wonder if you meant `style_format`. Also being a setting name it should be in backticks.

Reply:
Oops typo thanks for noticing.

Reply:
@spocke probably worth changing the PR title too, as I believe it's the default commit message when we squash & merge